src/main/java/network/brightspots/rcv/RawContestConfig.java

```diff
@@ -245,6 +266,56 @@ public boolean isExcluded() {
     public void setExcluded(boolean excluded) {
       this.excluded = excluded;
     }
+
+    /**
+     * A stream of all aliases (which is guaranteed to be unique) and the candidate name
+     * (which is not guaranteed to be unique, i.e. it may exist in the list twice)
+     *
+     * @return a stream containing the candidate name and all aliases, with no null elements
+     */
+    public Stream<String> createStreamOfNameAndAllAliases() {
+      List<String> otherNames = new ArrayList<>();
+      if (!isNullOrBlank(this.name)) {
+        otherNames.add(this.name);
+      }
+      if (!isNullOrBlank(this.code)) {
+        otherNames.add(this.code);
+      }
+
+      return Stream.concat(this.aliases.stream(), otherNames.stream());
+    }
+
+    /**
+     * For display purposes, get a semicolon-separated list of aliases, including the code.
+     *
+     * @return a potentially-empty string
+     */
+    public String getSemicolonSeparatedAliases() {
```

Comment:
I'm a little concerned about this. E.g. if a user includes a semicolon in an alias (on a single line), it's a confusing display in the table column. This will particularly become an issue when we allow editing of table cells again (which is P1 issue #590).

Hard to think of a way around this at the moment, other than banning the use of semicolons in aliases, or forcing it to use newlines in the column...

Reply:
I could use a less-common character (\cdot, `·`?), though semicolon has the benefit of making it easy to edit the table inline rather than hitting an edit button and having it populate the original textboxes. I'm happy to add a no-semicolon check when checking if names are valid.

I haven't fully thought through the design in #590: do you envision an inline edit, a modal, or populating the original create fields? It seems like the decision there would affect the design here.

Reply:
The way I kind of had it working before was double-clicking the cell would allow you to edit it and type right in there, then you had to press enter to commit it. Definitely open to other designs though. It'd probably actually be really cool if we had a modal pop up with the full text area to edit, but that would take some work.

Maybe we can go with a happy medium, like a pipe | since that's at least typeable on a standard keyboard and my gut tells me it's less likely to appear in an alias than a semicolon??

Alternatively, a no-semicolon check would work too. I'm happy either way. @chughes297 might have useful thoughts on this.

Reply:
I figured semicolon was unlikely to appear in a name, but happy to swap with a pipe, or leave as-is and use newlines with a full text area modal popup (or even inline textarea). Neither will be a huge lift, so happy to change this to whichever you prefer.

Reply:
The backend can save a list, so it won't have newline or semicolon separation -- just confirming that's okay?

Reply:
Yeah, that sounds perfect. Does that require code changes to this PR, or is it purely something you'd want to handle in future work around this?

Reply:
You're saying that it'll just use standard JSON list format, right?

Reply:
Yes, that's correct, and that is how it's saved today.